frontend/ucc_gui/src/containers/Projects/ActiveProject/Step_3/LearnNewSkill/LearnNewSkill.js

```diff
@@ -101,7 +101,7 @@ class LearnNewSkill extends React.Component {
         } catch (err) {
             console.log(err)
         }
-        return AxiosConfig.post('charityproject/LearnNewSkill', formData,
+        return AxiosConfig.put('charityproject/LearnNewSkill/', formData,
```

Comment:
What if there is no record present for that skill?How data would be saved when the user is trying to save it for the first time?

Reply:
There is a catch for this work correctly. On completion of challenge 2 when user selects a particular adventure to do,  we will create an empty record for that user in the adventure table with the respective userid and project id